### PR TITLE
docs: add missing core team members to README

### PR DESCRIPTION
**Pull request by kefranabg**
**Summary**

Add missing team members in README.

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by bencodezen** [comment]
@kefranabg Hope you don't mind me piggy backing off your PR! Thought it would be easier than a separate PR for someone to review later.

**Comment by kefranabg** [comment]
Woops! Sorry about that @bencodezen 😛

**Comment by bencodezen** [comment]
> Woops! Sorry about that @bencodezen 😛

It's all good! Haha

**Comment by edisdev** [comment]
Hi ! @kefranabg  , Can I add myself too to list? :)

File: .all-contributorsrc [hunk 1]
```diff
@@ -78,6 +78,70 @@
...
+      "avatar_url": "https://avatars0.githubusercontent.com/u/38957202?v=4",
+      "profile": "https://billychin.netlify.com/",
+      "contributions": [
+        "code"
+      ]
+    },
+    {
+      "login": "bencodezen",
+      "name": "Ben Hong",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/4836334?s=460&v=4",
```
Comment (sarahdayan) [inline .all-contributorsrc]:
```suggestion
      "avatar_url": "https://avatars3.githubusercontent.com/u/4836334?v=4",
```

File: .all-contributorsrc [hunk 2]
```diff
@@ -78,6 +78,70 @@
...
+        "code"
+      ]
+    },
+    {
+      "login": "vicbergquist",
+      "name": "Victoria Bergquist",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/25737281?v=4",
+      "profile": "https://twitter.com/vicbergquist",
+      "contributions": [
+        "code"
```
Comment (sarahdayan) [inline .all-contributorsrc]:
```suggestion
        "code",
        "ux"
```

File: .all-contributorsrc (hunk 1 above)
Comment (bencodezen) [reply .all-contributorsrc]: Thanks for the suggestion @sarahdayan! Quick question, what does that parameter do?

Comment (sarahdayan) [reply .all-contributorsrc]: It sets the size of the image. We don't necessarily want it, so we can get the best possible quality for the image (the biggest).

Comment (bencodezen) [reply .all-contributorsrc]: Ah. That makes sense. 💡 Thanks!

**Pull request merged**